pages/instructorAssessmentInstances/instructorAssessmentInstancesClientJS.ejs

```diff
@@ -301,6 +306,8 @@ function actionButtonFormatter(_value, row, index) {
         .attr('data-name', row.name)
         .attr('data-number', row.number)
         .attr('data-date-formatted', row.date_formatted)
+        .attr('data-group-name', row.group_name)
+        .attr('data-uid-list', row.uid_list)
```

Comment:
This `uid_list` is actually an array that is being forced into a string here. For consistency, I suggest:

```suggestion
        .attr('data-uid-list', row.uid_list?.join(', ') || 'empty')
```

This way you also don't need to add spaces to commas in the other function, and it will more properly handle groups with no students.